### PR TITLE
Adds in documentation for the non-Shopify version of back in stock - MERGE 1/4

### PR DESCRIPTION
**Pull request by Vector919**
[CO-2085](https://jira.braze.com/browse/CO-2085)

- Includes documentation for Catalogs back in stock for non-shopify use cases

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Back-in-stock notifications for Catalogs are currently in early access. Contact your account manager if you're interested in participating in this early access.
+{% endalert %}
+
+
+## How it works
+When the selected custom event is performed by a user, it can be used to create a back-in-stock subscription for a combination of that user and catalog item that it occurred for, if the item's inventory field is currently at 0. Once the item is back in stock, users can be notified through a Canvas.
+
+Back-in-stock notifications are determined by notification rules. Subscription statuses will be observed, so if the user has not opted in to notifications, they will be filtered out and will not receive notifications.
+
+Note that users are only subscribed for 90 days. If an item isn't back in stock within 90 days, the user is removed from the subscription. Braze will process up to 10 item updates per minute. This means if you update 11 items in one minute, only the first 10 items will trigger the back-in-stock notification.
```
Comment (davehensley) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md:23]: Is it confusing that we're using the word "subscription" to mean two different things here?

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Follow these steps to set up back-in-stock notifications in a specific catalog.
+
+1. Go to your catalog and click the **Settings** tab.
+2. Select the **Back in stock** toggle.
+3. If you have not previously configured your global back in stock settings, you will be prompted to set up the custom events and properties that will be used to trigger back in stock notifications:
+    <br>
+    - **Custom event for subscriptions** is the Braze custom event that will be used to subscribe a user for back in stock notifications. When this event occurs, the user that performed the event will be subscribed.
+    - **Custom event for unsubscribing** is the Braze custom event that will be used to unsubscribe a user from back in stock notifications.
+    - **Item ID Event property** is the property on the above custom event that will be used to determine the item for a back in stock subscription. This property on the custom event should contain an Item ID, that is present in a catalog. The custom event should also contain a `catalog_name` property, to specify which catalog this item is in.
```
Comment (davehensley) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md:35]: Maybe we should mention that they need to use the same property for subscribing/unsubscribing?

**Review by davehensley — APPROVED** [review]

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md (hunk 1 above)
Comment (Vector919) [reply _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md:23]: Yeah, that's a good point, this is basically referring to global notification preferences. I'm going to remove the first part of the sentence here and clarify which is which

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    - **Fallback Catalog** This is the catalog that will be used for the back in stock subscription, if there is no `catalog_name` property present on the custom event. 
+    <br>
+    After setting these values, click **Save** and continue to the catalog's back in stock settings page.
+    <br>
+
+4. Set your notification rule. There are two options: **Notify all subscribed users** and **Notify a certain number of users per 10 minutes**. <br> Selecting **Notify all subscribed users** notifies all customers who are waiting when the item is back in stock. **Notify a certain number of users per 10 minutes** notifies a specified number of customers per 10 minutes per item. Braze will notify the specified numbers of customers in 10-minute increments until there are no more customers to notify, or until the item goes out of stock.
```
Comment (raghavnarain3) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md:41]: notify per 10 minutes isn't accurate anymore

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Note that users are only subscribed for 90 days. If an item isn't back in stock within 90 days, the user is removed from the subscription. Braze will process up to 10 item updates per minute. This means if you update 11 items in one minute, only the first 10 items will trigger the back-in-stock notification.
+
+## Setting up back-in-stock notifications
+
+Follow these steps to set up back-in-stock notifications in a specific catalog.
+
+1. Go to your catalog and click the **Settings** tab.
+2. Select the **Back in stock** toggle.
+3. If the global back in stock settings, you will be prompted to set up the custom events and properties that will be used to trigger back in stock notifications:
```
Comment (davehensley) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/back_in_stock_notifications.md:31]: Looks like some extra words got deleted here.

**Review by shihanxiong — APPROVED** [review]

**Comment by lydia-xie** [comment]
@cla-bot check

**Pull request merged**